cocotb/regression.py

```diff
@@ -188,7 +188,7 @@ def _discover_tests() -> Iterable[Test]:
                         test = getattr(module, test_name)
                     except AttributeError:
                         _logger.error("Requested test %s wasn't found in module %s", test_name, module_name)
```

Comment:
Looks like it doesn't find this type of logging that uses `%s` but not the `%` operator.

Reply:
Well that is used by the logger so that is expected.

Reply:
Yes, it appears to leave loggers lazy evaluation alone.